src/bp/core/converse/converse-router.ts

```diff
@@ -2,13 +2,12 @@ import { Logger } from 'botpress/sdk'
 import { StandardError } from 'common/http'
 import { HTTPServer } from 'core/app/server'
 import { ConverseService } from 'core/converse'
+import { CustomRouter } from 'core/routers/customRouter'
```

Comment:
Maybe it's out of the scope of this PR, but I think we shouldn't forget to rename `customRouter` into `custom-router` (same goes for conditionalMiddleware).

Reply:
Agreed, i'm trying to keep PRs at a reasonable size so i'm not changing a ton of stuff at the same time. I'll attack the routers folder later